packages/vuetify/src/util/helpers.ts

```diff
@@ -596,6 +596,22 @@ export function focusChild (el: Element, location?: 'next' | 'prev' | 'first' |
   }
 }
 
+export function getTabbableChild (el: Element, location?: 'next' | 'prev') {
+  const targets = ['button', '[href]', 'input:not([type="hidden"])', 'select', 'textarea', '[tabindex]']
+    .map(s => `${s}:not([disabled])`)
+    .join(', ')
+  const children = [...el.querySelectorAll(targets)] as HTMLElement[]
+
+  let _el
+  let idx = children.indexOf(document.activeElement as HTMLElement)
+  const inc = location === 'next' ? 1 : -1
+  do {
```

Comment:
This might be worth abstracting into a shared function since it's basically this: https://github.com/vuetifyjs/vuetify/pull/17674/files#diff-6daf9735657f26888ecc2f70a2fa7b67dfef7600ab7275e0ae89b111dd461c6dR590

Reply:
abstracted into getNextElement()